clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -1983,6 +2009,10 @@ SubscriptionState subscriptions() {
         return subscriptions;
     }
 
+    FetchCommittedOffsetsEvent pendingOffsetFetch() {
```

Comment:
would `boolean hasPendingOffsetFetch()` be sufficient for the test?

Reply:
That made sense, so I changed it. Thanks!